lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -530,6 +545,17 @@ toApiNetworkParameters (NetworkParameters gp pp) = ApiNetworkParameters
     (Quantity $ unDecentralizationLevel $ view #decentralizationLevel pp)
     (view #desiredNumberOfStakePools pp)
     (Quantity $ fromIntegral $ getCoin $ view #minimumUTxOvalue pp)
+    (fromEraTransition <$> (view #transitionEras pp))
+  where
+    fromEraTransition (EraTransition info (Just epochN)) =
+        ApiEraTransition
+        (ApiEraTransitionInfo $ ApiT info)
+        (Just $ ApiEpochInfo
+            (ApiT epochN)
+            (Slotting.epochStartTime (Slotting.slotParams gp) epochN)
```

Comment:
Be aware of this: https://github.com/input-output-hk/cardano-wallet/pull/1913#discussion_r456778755